src/ucp/tag/tag_recv.c

```diff
@@ -179,11 +189,13 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_recv_nb,
                  ucp_tag_recv_callback_t cb)
 {
     ucp_request_param_t param = {
-        .op_attr_mask = UCP_OP_ATTR_FIELD_DATATYPE |
-                        UCP_OP_ATTR_FIELD_CALLBACK |
+        .op_attr_mask = UCP_OP_ATTR_FIELD_DATATYPE    |
+                        UCP_OP_ATTR_FIELD_CALLBACK    |
+                        UCP_OP_ATTR_FIELD_MEMORY_TYPE |
```

Comment:
no need to add UCP_OP_ATTR_FIELD_MEMORY_TYPE param, the default is already UNKNOWN